.github/workflows/downstream-project-spl.yml

```diff
@@ -134,11 +134,26 @@ jobs:
         with:
           version: "v0.9.1"
 
-      - shell: bash
+      - name: Install dependencies
+        shell: bash
         run: |
           source .github/scripts/downstream-project-spl-install-deps.sh
           source .github/scripts/downstream-project-spl-common.sh "${{ matrix.programs }}"
           if [ -n "$SKIP_SPL_DOWNSTREAM_PROJECT_TEST" ]; then
             exit 0
           fi
-          $CARGO_TEST_SBF --manifest-path program/Cargo.toml
+
+      - name: Test SBPFv0
+        shell: bash
+        run: |
+          source ci/downstream-projects/common.sh
+          cd "${{ matrix.programs }}"
+          $CARGO_TEST_SBF --arch v0 --manifest-path program/Cargo.toml
+
+      - name: Test SBPFv1
+        shell: bash
+        run: |
+          source ci/downstream-projects/common.sh
+          cd "${{ matrix.programs }}"
+          cargo clean
```

Comment:
Question: is this `cargo clean` needed? I thought all of the built artifacts would end up in different places for the different architectures, making this unnecessary. Or is this to save disk space on the runner?

Reply:
The increase in test time has come up before. It is not necessary to re-build the test, and the `cargo clean` is there as a convenience for erasing the SBPF related items.

In the `stable-sbf` job, I simply deleted the SBPF folders from target:

https://github.com/anza-xyz/agave/blob/75b72c0e36634093d544f7d391b26041ba4acada/programs/sbf/Makefile#L18-L19

I can replicate that in this PR, or use your approach.